packages/css-data/bin/mdn-data.ts

```diff
@@ -57,12 +37,62 @@ const normalizedValues = {
     value: 0,
     unit: "px",
   },
-  // https://github.com/mdn/data/issues/556
-  // @todo remove once fixed
   "background-size": autoValue,
   "text-size-adjust": autoValue,
 } as const;
 
+const parseInitialValue = (property: string, value: string): StyleValue => {
+  // Our default values hardcoded because no single standard
```

Comment:
any links to learn from reg discrepancies in the standard?

Reply:
It's not my comment. Probably yours. Just moved the code into function.

Reply:
damn, I wish I had cr back then, now I have no idea what I was looking at

Reply:
Well, you can see from issues. Some properties initials were different from browsers. Some just didn't work for input because of multiple values.